### ISSUE TITLE
Text Classification API (preview 0.20.0-preview.22313.1) Method not found

### ISSUE THREAD

**Issue by aforoughi1** [dotnet/machinelearning#6283]
**System Information (please complete the following information):**

    <PackageReference Include="Microsoft.Data.Analysis" Version="0.20.0-preview.22313.1" />
    <PackageReference Include="Microsoft.ML" Version="2.0.0-preview.22313.1" />
    <PackageReference Include="Microsoft.ML.AutoML" Version="0.20.0-preview.22313.1" />
    <PackageReference Include="Microsoft.ML.FastTree" Version="2.0.0-preview.22313.1" />
    <PackageReference Include="Microsoft.ML.LightGbm" Version="2.0.0-preview.22313.1" />
    <PackageReference Include="Microsoft.ML.TorchSharp" Version="0.20.0-preview.22313.1" />
    <PackageReference Include="TorchSharp-cpu" Version="0.97.3" />

**Describe the bug**
When I call Fit to train the model, I get 

System.MissingMethodException
  HResult=0x80131513
  Message=Method not found: 'Void Module.train()'.
  Source=Microsoft.ML.TorchSharp
  StackTrace:
   at Microsoft.ML.TorchSharp.NasBert.TextClassificationTrainer.Trainer..ctor(TextClassificationTrainer parent, IChannel ch)
   at Microsoft.ML.TorchSharp.NasBert.TextClassificationTrainer.Fit(IDataView input)
   at Microsoft.ML.Data.EstimatorChain`1.Fit(IDataView input)

**To Reproduce**

using Microsoft.ML;
using Microsoft.ML.AutoML;
using Microsoft.ML.Data;
using Microsoft.ML.TorchSharp;
using System.Linq;
using System.Threading.Tasks;

    public class SentimentData
    {
        [LoadColumn(0)]
        public string Sentence { get; set; }

        //Sentiment is a string: 'positive', 'negative' or 'neutral'
        [LoadColumn(1)]
        public string Sentiment { get; set; }

        // Label = {0:'neutral', 1:'positive',-1:'negative'}
        [LoadColumn(2)]
        public float Label { get; set; }
    }

    class Program
    {
        static async Task Main(string[] args)
        {
            MLContext mlContext = new MLContext(seed: null);
            IDataView data = mlContext.Data.LoadFromTextFile<SentimentData>(@"..\..\..\FinancialPhraseBank.txt", separatorChar: '\t', hasHeader: true);
            var preview = data.Preview(6000);

            var trainTestSplit = mlContext.Data.TrainTestSplit(data, testFraction: 0.2);
            var trainSet = trainTestSplit.TrainSet;
            var validationSet = trainTestSplit.TestSet;

            //Sentence is input
            //Sentiment is a string: 'positive', 'negative' or 'neutral'
            //Label = {0:'neutral', 1:'positive',-1:'negative'}

            var pipeline = mlContext.Transforms.Conversion.MapValueToKey(outputColumnName: "KeyColumn", "Label")
                                     .Append(mlContext.MulticlassClassification.Trainers.TextClassification(3, labelColumnName: "KeyColumn", sentence1ColumnName: "Sentence"));

            //it crashes here
            var model = pipeline.Fit(trainSet);
        }
    }
**Additional context**

[FinancialPhraseBank.txt](https://github.com/dotnet/machinelearning/files/9351179/FinancialPhraseBank.txt)

**Comment by dakersnar** [comment]
Could you please fix the formatting on your code example? I can try to reproduce this bug locally.

**Comment by aforoughi1** [comment]
I think its a duplicate bug #626
Attached a test project
[TestAutoMlApi.zip](https://github.com/dotnet/machinelearning/files/9401534/TestAutoMlApi.zip)
3

**Comment by dakersnar** [comment]
Thank you for the additional repro information. I'm going to close this issue as a duplicate, as the other one can continue to track it.
